src/site/assignments/country-catalogue/country-catalogue.rst

```diff
@@ -123,63 +123,148 @@ instances of the ``Country`` class.
 Part 2 --- Country Catalogue
 ============================
 
-The ``CountryCatalogue`` class will be a collection of ``Country`` objects.
+The ``CountryCatalogue`` class holds references to ``Country`` objects. Additionally, the ``CountryCatalogue`` provides
+functionality to add and remove ``Country`` objects to itself and ask questions about the data. The ``CountryCatalogue``
+is effectively a *list* keeping track of the ``Country`` objects with additional functionality.
 
-There will be two fields
+Although the description of this class is provided with an order and each part is numbered, one should feel free to
+complete the methods in any order they see fit. Additionally, feel free to write and use additional private methods as
```

Comment:
> Unlike previous years, I am removing explicit mention of private methods

:thinking:

Reply:
You think it's too much to leave out?

Reply:
I think it's fine either way. It's just counter to your description.

Reply:
I see what you mean. What I meant was, previously I would say "add a private method to expand capacity/shuffle the contents of the array/find an element/etc.". Now I just say _yo, if you want them, have at it_.